package.json

```diff
@@ -77,28 +77,30 @@
     "url": "https://github.com/elastic/kibana.git"
   },
   "resolutions": {
-    "**/@types/node": ">=10.17.17 <10.20.0",
-    "**/@types/react": "^16.9.36",
-    "**/@types/hapi": "^17.0.18",
     "**/@types/angular": "^1.6.56",
-    "**/@types/hoist-non-react-statics": "^3.3.1",
     "**/@types/chai": "^4.2.11",
+    "**/@types/hapi": "^17.0.18",
+    "**/@types/hoist-non-react-statics": "^3.3.1",
+    "**/@types/node": ">=10.17.17 <10.20.0",
+    "**/@types/react": "^16.9.36",
+    "**/cross-fetch/node-fetch": "^2.6.1",
     "**/cypress/@types/lodash": "^4.14.159",
     "**/cypress/lodash": "^4.17.20",
-    "**/typescript": "4.0.2",
+    "**/deepmerge": "^4.2.2",
+    "**/fast-deep-equal": "^3.1.1",
     "**/graphql-toolkit/lodash": "^4.17.15",
     "**/hoist-non-react-statics": "^3.3.2",
+    "**/image-diff/gm/debug": "^2.6.9",
+    "**/isomorphic-fetch/node-fetch": "^2.6.1",
     "**/isomorphic-git/**/base64-js": "^1.2.1",
     "**/istanbul-instrumenter-loader/schema-utils": "1.0.0",
-    "**/image-diff/gm/debug": "^2.6.9",
     "**/load-grunt-config/lodash": "^4.17.20",
     "**/node-jose/node-forge": "^0.10.0",
-    "**/react-dom": "^16.12.0",
     "**/react": "^16.12.0",
+    "**/react-dom": "^16.12.0",
     "**/react-test-renderer": "^16.12.0",
     "**/request": "^2.88.2",
-    "**/deepmerge": "^4.2.2",
-    "**/fast-deep-equal": "^3.1.1"
+    "**/typescript": "4.0.2"
```

Comment:
Sorry for the messy diff. I sorted the list alphabetically because it was all over the place. The only change I made was adding these two lines:

```diff
+    "**/cross-fetch/node-fetch": "^2.6.1",
+    "**/isomorphic-fetch/node-fetch": "^2.6.1",
```